test/kernel-headers/source/runners/0-setup.sh

```diff
@@ -16,35 +19,45 @@ then
     exit 1
 fi
 
+
 # don't do this until after vagrant up so the script can check the result and cleanup and exit if it fails
 source "${EBPF_NET_SRC_ROOT}/dev/script/bash-error-lib.sh"
 
-# Check if local docker registry is running
-result=$(docker ps | grep registry:latest | grep local-docker-registry) || true
-if [[ "${result}" == "" ]]
-then
-  # local registry isn't running, so start it
-  "${EBPF_NET_SRC_ROOT}/dev/docker-registry.sh"
-fi
+vagrant plugin install vagrant-sshfs
 
-# Check that required docker images exist
-result1=$(docker images | grep ^kernel-collector) || true
-result2=$(docker images | grep ^reducer) || true
-if [[ "${result1}" == "" || "${result2}" == "" ]]
+vagrant plugin install vagrant-scp
```

Comment:
Same - could this be done in the runner setup, not each time this script is run?